vendor/bat-native-ledger/src/bat/ledger/internal/publisher/publisher_status_helper.h

```diff
@@ -0,0 +1,32 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVELEDGER_PUBLISHER_PUBLISHER_STATUS_HELPER_H_
+#define BRAVELEDGER_PUBLISHER_PUBLISHER_STATUS_HELPER_H_
+
+#include "bat/ledger/ledger.h"
+
+namespace bat_ledger {
+class LedgerImpl;
+}
+
+namespace braveledger_publisher {
+
+// Refreshes the publisher status for each entry in the specified list
+void RefreshPublisherStatus(
```

Comment:
These functions are used when a list of records is returned from the database that includes the publisher status. For those queries, the `server_publisher_info` table is linked by left join. Now that we are using the `server_publisher_info` table as a cache, it may be missing data, or it may have outdated data.

These functions are used by queries that need up-to-date publisher status information to update the publisher status for each entry in the list before calling the supplied callback.